examples/graph/app.js

```diff
@@ -162,34 +163,51 @@ class Root extends Component {
     }
   }
 
-  _onDragMove(el) {
+  _onMouseDown(event) {
+    // Use DeckGL.queryObject() to find the object under the mouse,
+    // and store it for updating on mouse move.
+    const point = {x: event.clientX, y: event.clientY};
+    const el = this.deckGL.queryObject(point);
     if (el) {
-      const {viewport} = this.state;
-      const {width, height} = viewport;
-      const x = el.x - width / 2;
-      const y = el.y - height / 2;
       this.setState({
         clicked: el,
         dragging: {
-          node: el.object,
-          x,
-          y
+          node: el.object
         }
       });
+      this._updateDraggedElement(point);
     }
   }
 
-  _onDragEnd(el) {
-    if (el) {
-      this.setState({
-        dragging: null,
-        lastDragged: {
-          node: el.object
-        }
-      });
+  _onMouseMove(event) {
+    if (this.state.dragging) {
+      const point = {x: event.clientX, y: event.clientY};
```

Comment:
I can change it in the example, but note that picking uses objects with `x` and `y` props (supersets of points so not exactly apples-to-apples but that's where it came from in this example).